dsl/pace/dsl/dace/build.py

```diff
@@ -170,7 +170,9 @@ def get_sdfg_path(
             )
         # Check resolution per tile
         build_resolution = ast.literal_eval(build_info_file.readline())
-        if config.tile_resolution != build_resolution:
+        if (config.tile_resolution[0] / config.layout[0]) != (
+            build_resolution[0] / build_layout[0]
+        ):
             raise RuntimeError(
```

Comment:
This change is probably fine, but just curious - what's the rationale behind it? Was this an existing bug?

Reply:
It was a bug due to a bad multi-branch merge. Basically tile_resolution is decomposition dependent (e.g. 6 ranks = 192, 54 ranks = 576) but what is inlined in dace is the loop sizes, e.g. 192. So you have to divide to see if those match